README.md

```diff
@@ -0,0 +1,137 @@
+# Aligent Magento Integration and API Functional Testing Pipe
+
+A bitbucket pipe for running Magento integration and api functional tests 
+
+It is designed to be run parallelly so you can leverage bitbucket parallel steps. [Example pipeline](#example-pipeline)
+
+The pipe detects a `composer.lock` file and installs packages. If no `composer.lock` file is found, the pipe will create
+a new magento project.
+
+## Environment Variables
+
+| Variable              | Usage                                                                                                                                                      |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `TYPE`                  | Available tests are `integration`, `rest`, `graphql`. Default: `integration`                                                       |
+| `DATABASE_USERNAME`     | Default: `user`                                                                                                                                            |
+| `DATABASE_ROOTPASSWORD` | Default: `rootpassword`                                                                                                                                    |
+| `DATABASE_PASSWORD`     | Default: `password`                                                                                                                                        |
+| `ELASTICSEARCH_HOST`    | Must be `host.docker.internal` when running in pipelines. Optionally change this if you are developing the pipe locally.                                   |
+| `RABBITMQ_HOST`         | Must be `host.docker.internal` when running in pipelines. Optionally change this if you are developing the pipe locally.                                   |
+| `DATABASE_HOST`         | Must be `host.docker.internal` when running in pipelines. Optionally change this if you are developing the pipe locally.                                   |
+| `COMPOSER_AUTH`         | JSON stringified composer `auth.json` with the relevant configuration you need.                                                                            |
+| `REPOSITORY_URL`        | `https://repo.magento.com/` - If using this, make sure the `COMPOSER_AUTH` variable is set. <br>  `https://mirror.mage-os.org/` - Only supports open source edition. <br> Default: `https://repo.magento.com/` |
+| `MAGENTO_VERSION`       | (Optional) Default: `magento/project-community-edition:>=2.4.5 <2.4.6`                                                                                     |
```

Comment:
Can give an example for commerce here too?

Reply:
I've added this